src/pages/ValidatorManagement/ManageValidator/DepositStepper.tsx

```diff
@@ -0,0 +1,136 @@
+import { Stepper, Step } from 'react-form-stepper'
+import { useNavigate } from 'react-router-dom'
+import { DepositStatusSteps, FORM_STEPS } from '../../../constants'
+import { useWindowSize } from '../../../hooks/useWindowSize'
+import styles from './DepositStepper.module.css'
+type DepositStepperProps = {
+  activeStep: number
+}
+
+const DepositStepper = ({ activeStep }: DepositStepperProps) => {
+  const navigate = useNavigate()
+  const windowSize = useWindowSize()
+
+  const stepToUrlMap = [
+    '/validator-management/',
+    // fix paths
+  ]
+
+  const getIsStepVisible = (
```

Comment:
Can you clarify the purpose of this?

Reply:
@EmilIvanichkovv As you can see I have left a comment that I have to fix it. In the last meet we discussed with Aqeel that I have to wait for further instructions. I still don't know how this stepper should work.

Reply:
I am referring to `getIsStepVisible`. The “math” in there seems kinda odd.